lib/stitches/generator_files/spec/features/api_spec.rb

```diff
@@ -80,7 +80,7 @@
       correct_code && correct_header
     end
 
-    failure_message_for_should do
+    failure_message do
```

Comment:
We should, but it has to be in the gem spec if it's a real requirement. Gentile.lock is irrelevant for determining transitive dependencies. 

Dave

(Sent from my iPhone [I CAN spell])

> On Jul 9, 2015, at 6:54 PM, Andrew Peterson <notifications@github.com> wrote:
> 
> In lib/stitches/generator_files/spec/features/api_spec.rb:
> 
> > @@ -80,7 +80,7 @@
> >        correct_code && correct_header
> >      end
> >  
> > -    failure_message_for_should do
> > +    failure_message do
> It requires rspec-rails3, which looks like it is rspec3, if I'm ready Gemfile.lock's crypticness right.
> 
> Any reason not to require rspec 3? Monthly bundle update and all that.
> 
> —
> Reply to this email directly or view it on GitHub.
>